geocities.jp,"HTTPSConnectionPool(host='geocities.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af8b205b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='geocities.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af8b20e80>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
geomedian.com,"HTTPSConnectionPool(host='geomedian.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='geomedian.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
gesetze-im-internet.de,"HTTPSConnectionPool(host='gesetze-im-internet.de', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af7ffa9d0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='gesetze-im-internet.de', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af79c05b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gestionderiesgos.gob.ec,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
gesundheitsinformation.de,"HTTPSConnectionPool(host='gesundheitsinformation.de', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af78d2130>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='gesundheitsinformation.de', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af74507c0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gets.govt.nz,"HTTPSConnectionPool(host='www.gets.govt.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gets.govt.nz', port=443): Read timed out. (read timeout=2)"
geva.co.il,"HTTPSConnectionPool(host='geva.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.geva.co.il', port=443): Read timed out. (read timeout=2)"
ghafla.com,Exceeded 30 redirects.,Exceeded 30 redirects.
ghanely.me,"HTTPSConnectionPool(host='ghanely.me', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1131)')))","HTTPSConnectionPool(host='simcast.com', port=443): Max retries exceeded with url: /?s=PLE&d=ghanely.me&ref=&rid=207 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
ghidulprimariilor.ro,"HTTPSConnectionPool(host='ghidulprimariilor.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='ghidulprimariilor.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
gids.tv,"HTTPSConnectionPool(host='www.gids.tv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gids.tv', port=443): Read timed out. (read timeout=2)"
gifu.lg.jp,"HTTPSConnectionPool(host='gifu.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af4a09340>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='gifu.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af8812c70>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gigabaza.ru,"HTTPSConnectionPool(host='gigabaza.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='gigabaza.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
girolami.eu,"HTTPSConnectionPool(host='girolami.eu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='girolami.eu', port=443): Read timed out. (read timeout=2)"
gismeteo.lt,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gismeteo.lv,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gismeteo.pl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gismeteo.ru,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gismeteo.ua,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
giuntialpunto.it,"HTTPSConnectionPool(host='giuntialpunto.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'giuntialpunto.it' doesn't match either of '*.giunti.it', 'giunti.it'"")))","HTTPSConnectionPool(host='www.giuntialpunto.it', port=443): Read timed out. (read timeout=2)"
givatayim.muni.il,"HTTPSConnectionPool(host='givatayim.muni.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5af87f9ca0>, 'Connection to givatayim.muni.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='givatayim.muni.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5af89aa100>, 'Connection to givatayim.muni.il timed out. (connect timeout=2)'))"
gizzmo.si,"HTTPSConnectionPool(host='gizzmo.si', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='gizzmo.si', port=443): Read timed out. (read timeout=2)"
glasistre.hr,"HTTPSConnectionPool(host='glasistre.hr', port=443): Read timed out.","HTTPConnectionPool(host='glasistre.hr', port=80): Read timed out."
glasnews.bg,"HTTPSConnectionPool(host='glasnews.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='glasnews.bg', port=443): Read timed out. (read timeout=2)"
glassumadije.rs,"HTTPSConnectionPool(host='www.glassumadije.rs', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.glassumadije.rs', port=80): Read timed out. (read timeout=2)"
glimp.co.nz,"HTTPSConnectionPool(host='www.glimp.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.glimp.co.nz', port=443): Read timed out. (read timeout=2)"
gme.cz,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gmposts.com,"HTTPSConnectionPool(host='gmposts.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='gmposts.com', port=80): Read timed out. (read timeout=2)"
gnr.pt,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
gobiernodecanarias.org,"HTTPSConnectionPool(host='gobiernodecanarias.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af69bb100>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='gobiernodecanarias.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af3ee9ac0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gogov.ru,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
goias.gov.br,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
golos.com.ua,"HTTPSConnectionPool(host='golos.com.ua', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af5c0bf10>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='www.golos.com.ua', port=80): Read timed out. (read timeout=2)"
gonews.it,"HTTPSConnectionPool(host='www.gonews.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gonews.it', port=443): Read timed out. (read timeout=2)"
goodiz.tv,"HTTPSConnectionPool(host='goodiz.tv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.goodiz.tv', port=443): Read timed out. (read timeout=2)"
goodreads.com,"HTTPSConnectionPool(host='goodreads.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.goodreads.com', port=443): Read timed out. (read timeout=2)"
goodsongs.com.ua,"HTTPSConnectionPool(host='goodsongs.com.ua', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='goodsongs.com.ua', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
googleusercontent.com,"HTTPSConnectionPool(host='googleusercontent.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af7964820>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='googleusercontent.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af626a880>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gorzow.pl,"HTTPSConnectionPool(host='gorzow.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af1b3c850>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='gorzow.pl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af33c5d30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gourmet.cl,"HTTPSConnectionPool(host='www.gourmet.cl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gourmet.cl', port=443): Read timed out. (read timeout=2)"
gov.bc.ca,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gov.hu,"HTTPSConnectionPool(host='gov.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af7285d60>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='gov.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af7285670>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
govextra.gov.il,"HTTPSConnectionPool(host='govextra.gov.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='govextra.gov.il', port=443): Read timed out. (read timeout=2)"
gpoint.co.jp,"HTTPSConnectionPool(host='gpoint.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'gpoint.co.jp' doesn't match '*.gpoint.co.jp'"")))","HTTPSConnectionPool(host='gpoint.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'gpoint.co.jp' doesn't match '*.gpoint.co.jp'"")))"
grad-zadar.hr,"HTTPSConnectionPool(host='grad-zadar.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af664f0a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='grad-zadar.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af6c63d60>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gramatnicaglobuss.lv,"HTTPSConnectionPool(host='gramatnicaglobuss.lv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gramatnicaglobuss.lv', port=443): Read timed out. (read timeout=2)"
grazia.it,"HTTPSConnectionPool(host='grazia.it', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5af2c65940>, 'Connection to grazia.it timed out. (connect timeout=2)'))","HTTPConnectionPool(host='grazia.it', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5af716a670>, 'Connection to grazia.it timed out. (connect timeout=2)'))"
graziadaily.co.uk,"HTTPSConnectionPool(host='graziadaily.co.uk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='graziadaily.co.uk', port=443): Read timed out. (read timeout=2)"
grenka.ua,"HTTPSConnectionPool(host='grenka.ua', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='grenka.ua', port=443): Read timed out. (read timeout=2)"
grizly.cz,"HTTPSConnectionPool(host='www.grizly.cz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.grizly.cz', port=443): Read timed out. (read timeout=2)"
gruene.at,"HTTPSConnectionPool(host='gruene.at', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af2e74580>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='gruene.at', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af8217c70>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
grupolusofona.pt,"HTTPSConnectionPool(host='grupolusofona.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af6558d30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='grupolusofona.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5aef9fc700>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gsb.or.th,"HTTPSConnectionPool(host='gsb.or.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af3833730>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='gsb.or.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5af7e43220>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gtp.gr,"HTTPSConnectionPool(host='gtp.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'gtp.gr' doesn't match either of 'www.gtp.gr', 'news.gtp.gr', 'traveloffers.gtp.gr', 'jobs.gtp.gr', 'about.gtp.gr', 'ferries.gtp.gr'"")))","HTTPSConnectionPool(host='gtp.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'gtp.gr' doesn't match either of 'www.gtp.gr', 'news.gtp.gr', 'traveloffers.gtp.gr', 'jobs.gtp.gr', 'about.gtp.gr', 'ferries.gtp.gr'"")))"
guaranteepra.com,"HTTPSConnectionPool(host='guaranteepra.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5af7461b20>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='guaranteepra.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5aef66a9a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
guarden.co,"HTTPSConnectionPool(host='guarden.co', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='guarden.co', port=80): Read timed out. (read timeout=2)"
guiadelocio.com,"HTTPSConnectionPool(host='www.guiadelocio.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.guiadelocio.com', port=443): Read timed out. (read timeout=2)"
guiagt.com,"HTTPSConnectionPool(host='guiagt.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='guiagt.com', port=443): Read timed out. (read timeout=2)"
guiato.com.br,"HTTPSConnectionPool(host='guiato.com.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='guiato.com.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
gunadarma.ac.id,"HTTPSConnectionPool(host='gunadarma.ac.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.gunadarma.ac.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
guru.ua,"HTTPSConnectionPool(host='guru.ua', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5af83bf970>, 'Connection to guru.ua timed out. (connect timeout=2)'))","HTTPConnectionPool(host='guru.ua', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5af7f0a160>, 'Connection to guru.ua timed out. (connect timeout=2)'))"
gustissimo.it,"HTTPSConnectionPool(host='gustissimo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.gustissimo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
guzzle.co.za,"HTTPSConnectionPool(host='guzzle.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'guzzle.co.za' doesn't match 'www.guzzle.co.za'"")))","HTTPSConnectionPool(host='www.guzzle.co.za', port=443): Read timed out. (read timeout=2)"
gyorplusz.hu,"HTTPSConnectionPool(host='www.gyorplusz.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gyorplusz.hu', port=443): Read timed out. (read timeout=2)"
h-alter.org,"HTTPSConnectionPool(host='h-alter.org', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='h-alter.org', port=80): Read timed out. (read timeout=2)"
ha.rs,"HTTPSConnectionPool(host='ha.rs', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5aef7b4a30>, 'Connection to ha.rs timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ha.rs', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5af63781f0>, 'Connection to ha.rs timed out. (connect timeout=2)'))"
haifa.ac.il,"HTTPSConnectionPool(host='haifa.ac.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='haifa.ac.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
hakolhayehudi.co.il,"HTTPSConnectionPool(host='www.hakolhayehudi.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='hakolhayehudi.co.il', port=80): Read timed out."
hamazo.tv,"HTTPSConnectionPool(host='hamazo.tv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.hamazo.tv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
